pkg/cache/v3/cache.go

```diff
@@ -224,7 +225,7 @@ func (r *RawDeltaResponse) GetDeltaDiscoveryResponse() (*discovery.DeltaDiscover
 		marshaledResources := make([]*discovery.Resource, len(r.Resources))
 
 		for i, resource := range r.Resources {
-
+			name := GetResourceName(resource)
```

Comment:
You mean pulling from `ResourceNamesSubscribed()`? I think that would work with the exception of wildcard because then no names would exist in that field

Reply:
I think I got this confused with the type url, this is fine as is. Essentially I was looking for a way to get the resource name without having to know how to handle the specific type (to better support unknown resource types), but since we're tackling that in a follow up I think we can leave this

Reply:
Cool. Yup I'll revisit this when we add support for custom resource types.